wgpu-hal/src/gles/adapter.rs

```diff
@@ -666,10 +665,10 @@ impl crate::Adapter<super::Api> for super::Adapter {
         let empty = Tfc::empty();
         let base = Tfc::COPY_SRC | Tfc::COPY_DST;
         let unfilterable = base | Tfc::SAMPLED;
-        let depth = base | Tfc::SAMPLED | Tfc::MULTISAMPLE | Tfc::DEPTH_STENCIL_ATTACHMENT;
+        let depth = base | Tfc::SAMPLED | Tfc::MULTISAMPLE_X4 | Tfc::DEPTH_STENCIL_ATTACHMENT;
```

Comment:
We need to get glow::MAX_SAMPLES with `gl.get_parameter_i32` and then use the value to populate if 2, 4, or 8 is available.

Reply:
I read through the GL doc but I still have a question :
does gl.get_parameter_i32() value means that smaller sampling count is supported? , for example : 
if it returns 32 , does that means that 16,8 etc is supported ? .

Reply:
Correct